src/media/media.ts

```diff
@@ -447,6 +447,11 @@ export class SdkMedia extends (EventEmitter as { new(): StrictEventEmitter<Event
       createAndEmitSdkError.call(this.sdk, SdkErrorTypes.media, e);
       throw e;
     });
+    stream.getTracks().forEach(track => {
```

Comment:
We don't want to remove the audio track if its muted, only the video track. If I remember correctly this issue was because we were getting two video tracks, one muted and one not.

Reply:
Oh true. For some reason I thought it was for both kinds.  I'll fix that now